_rules/SC1-2-video-transcript.md

```diff
@@ -101,10 +94,6 @@ Then he stops to scratch his bottom.</p>
 
 ## Inapplicable
 
```

Comment:
Include an inapplicable test case for streaming media

Reply:
Similar to previous comment. We didn't know how to do it.